cypress/e2e/test.spec.ts

```diff
@@ -0,0 +1,73 @@
+import basketPage from "../pages/basketPage";
+import modifyOrderPage from "../pages/modifyOrderPage";
+import orderNavigationPage from "../pages/orderNavigationPage";
+import orderPastaPage from "../pages/orderPastaPage";
+import orderPizzaPage from "../pages/orderPizzaPage";
+
+describe("Testing pizza hut application", () => {
+	before(() => {
+		orderNavigationPage.pizzaMenuButton.click();
+	});
+
+	it("Should select and verify item", () => {
+		orderPizzaPage.pizzaCards.eq(0)
+			.within(() => {
+				orderPizzaPage.pizzaNameInCard
+					.then($element => {
+						cy.wrap($element.contents()[0].textContent).as("pizzaName")
+					});
+				orderPizzaPage.pizzaPriceInCard
+					.invoke("text")
+					.as("pizzaPrice");
+				orderPizzaPage.pizzaAddButtonInCard
+					.click();
+			});
```

Comment:
Not clear what is happening here. Please explain

Reply:
Would appreciate if added few comments for steps or use reporting to address these steps

Reply:
Done